recipe/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "python-poppler" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/15/ff/e9c8f176c376223146d212771016595b4ad8d0a83d40636b7b39798a7219/python_poppler-{{ version }}.tar.gz
+  sha256: e6ca5c23ec02350bf2cef85a6bf9f1b261796436db478176f9d24fb14eeecc6a
+
+build:
+  number: 0
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv # [unix]
+  
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - cmake >=3.18
+    - ninja       # [win]
+    - pkg-config
+    
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+    - meson >=1.0.0
```

```suggestion
    - meson >=1.1.0
```
https://github.com/cbrunet/python-poppler/blob/v0.4.1/meson.build#L7